util/fpga/cw_spiflash.py

```diff
@@ -0,0 +1,273 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Utilities used to download firmware images into the OpenTitan system."""
+
+import time
+import hashlib
+from functools import partial
+
+
+def _uint_to_le(val, length):
+    """Returns a byte array that represents an unsigned integer in little-endian format.
+
+    Args:
+      val: Unsigned integer to convert.
+      length: Number of bytes.
+
+    Returns:
+      A byte array of ``length`` bytes that represents ``val`` in little-endian format.
+    """
+    return val.to_bytes(length=length, byteorder='little')
+
+
+def _le_to_uint(val):
+    """Returns the unsigned integer represented by the given byte array in little-endian format.
+
+    Args:
+      val: Byte array in little-endian format that represents an unsigned integer.
+
+    Returns:
+      The unsigned integer represented by the byte array ``val``.
+    """
+    return int.from_bytes(val, byteorder='little')
```

Comment:
These are only used a couple of times. Inline them?

Reply:
IIRC the main reason was just to make the f-string below shorter (`_le_to_uint`), `_uint_to_le` just for consistency.